.github/workflows/generate_manpage.yml

```diff
@@ -1,7 +1,7 @@
 name: Generate Manpages
 
 on:
-  push:
+  pull_request:
```

Comment:
@SpyTec, do you think it is worth to keep the pull_request event and the `push` one in the same workflow? 

My thought here is more like: "What if we push a new dependency and forget about it and the workflow doesn't work anymore? This way we would see right in the PR that something changed". 

But since we have it to be executed in the main branch after every push, I think it will be more a question of checking it out before releasing rather than looking at it for every PR?

Reply:
I think we can keep whichever is best, so if pull_request is better we can keep this change

Reply:
@r0x0d Do you think I should leave it pull_request?

Reply:
Lets leave as pull_request for now. I think it will be better to look at it constantly rather than when pushing on main-basis. I don't know if anyone keeps an eye in main pipelines.